pipelines/release/tarball.groovy

```diff
@@ -598,22 +598,32 @@ def void writeScript(Map p) {
  * Push {@code ./distrib} dir to an s3 bucket under the "path" formed by
  * joining the {@code parts} parameters.
  */
-def void s3PushDocker(String ... parts) {
+def void s3PushConda(String ... parts) {
   def objectPrefix = "stack/" + util.joinPath(parts)
   def cwd = pwd()
+  def buildDir = "${cwd}/build"
 
   def env = [
     "EUPS_PKGROOT=${cwd}/distrib",
     "EUPS_S3_OBJECT_PREFIX=${objectPrefix}",
     "HOME=${cwd}/home",
+    "BUILDDIR=${buildDir}"
   ]
 
   withEnv(env) {
     withEupsBucketEnv {
       timeout(time: 10, unit: 'MINUTES') {
-        docker.image(util.defaultAwscliImage()).inside {
+        //docker.image(util.defaultAwscliImage()).inside {
           // alpine does not include bash by default
-          util.posixSh(s3PushCmd())
+        util.posixSh("""
+        source "${BUILDDIR}/conda/miniconda3-py38_4.9.2/etc/profile.d/conda.sh"
+        conda create --name aws-cli-env
+        conda activate aws-cli-env
+        conda install pip
+        pip install awscli
```

Comment:
I guess `conda create --name aws-cli-env` automatically replaces an environment if it already exists.  This is a bit wasteful, but it does update `awscli` to the latest version every time, which is usually desirable.  The alternative would be to look for the environment and upgrade it if it exists.

I think this can still be more efficient by using `mamba`, avoiding `pip`, and doing things in one step, so maybe something like this?
```
if conda env list | grep aws-cli-env > /dev/null 2>&1; then
    conda activate aws-cli-env
    mamba update awscli
else
    mamba create -y --name aws-cli-env awscli
    conda activate aws-cli-env
fi
```